charts/agent/templates/_config-exporters.tpl

```diff
@@ -1,13 +1,13 @@
 {{- define "config.exporters.otlphttp.observe" -}}
 otlphttp/observe:
-    endpoint: "{{ .Values.observe.collectionEndpoint }}v2/otel"
+    endpoint: "{{ .Values.observe.collectionEndpoint }}/v2/otel"
```

Comment:
I have a tweak for this section that makes it so it doesn't matter if endpoint comes in with slash or not.  I can rebase on this to reconcile.